tests/test_filter_tasks.py

```diff
@@ -141,3 +141,84 @@ def test_filter_tasks_exclude_aggregate(
         if exclude_aggregate:
             # Aggregate tasks should be excluded
             assert not task.is_aggregate
+
+
+def test_filter_tasks_exclude_aggregate_with_task_classes():
+    """Test that filter_tasks correctly handles aggregate tasks when passed task classes.
+
+    This is a regression test for a bug where accessing t.is_aggregate on a task class
+    would fail because is_aggregate is a property that requires an instance.
+    """
+    from mteb.abstasks.aggregated_task import AbsTaskAggregate
+    from mteb.get_tasks import TASK_LIST
+
+    # Get task classes (not instances)
+    task_classes = list(TASK_LIST)
+
+    # Find at least one aggregate task class
+    aggregate_classes = [t for t in task_classes if issubclass(t, AbsTaskAggregate)]
+    assert len(aggregate_classes) > 0, "Expected at least one aggregate task class"
+
+    # Filter with exclude_aggregate=True
+    filtered = filter_tasks(task_classes, exclude_aggregate=True)
+
+    # Verify no aggregate tasks in result
+    for task_cls in filtered:
+        assert not issubclass(task_cls, AbsTaskAggregate), (
+            f"{task_cls.__name__} is an aggregate task but was not filtered out"
+        )
+
+    # Filter with exclude_aggregate=False should include aggregate tasks
+    filtered_with_aggregate = filter_tasks(task_classes, exclude_aggregate=False)
+    assert len(filtered_with_aggregate) >= len(filtered), (
+        "Expected more tasks when including aggregate tasks"
+    )
+
+    # Check that aggregate tasks are in the unfiltered result
+    aggregate_in_result = [
+        t for t in filtered_with_aggregate if issubclass(t, AbsTaskAggregate)
+    ]
+    assert len(aggregate_in_result) > 0, (
+        "Expected aggregate tasks when exclude_aggregate=False"
+    )
+
+
+def test_invalid_language_code():
+    """Test that invalid language codes raise an error."""
+    from mteb import get_tasks
+
+    with pytest.raises(ValueError, match="Invalid language code"):
+        get_tasks(languages=["invalid_lang"])
+
+
+def test_invalid_script_code():
+    """Test that invalid script codes raise an error."""
+    from mteb import get_tasks
+
+    with pytest.raises(ValueError, match="Invalid script code"):
+        get_tasks(script=["InvalidScript"])
```

Comment:
these are tests for get_tasks, they shouldn't be here. There is also no need to import the function inside the test